app/ui/browser2/index.jsx

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
A lot of stuff in index.jsx looks like it could be shared between browser frontends. More meta, communication with the user agent service and handling of messages/diffs etc. should be dried up.